mysql-test/main/json_normalize.test

```diff
@@ -1,3 +1,6 @@
+# Test doesn't work on macOS.
```

Comment:
Minor suggestion:
This is redundant as including `not_mac.inc` already implies this.
Perhaps a brief explanation of why this test doesn't work on macOS as a comment or in the commit description would be better.

Apart from that, looks good.

Reply:
Hi Tony, thank you for the suggestion.  I haven't (yet) investigated why this test doesn't work on macOS, but I will.  Just not for this commit, as it's purpose is to demonstrate how to use `not_mac.inc`.